biogeochem/EDLoggingMortalityMod.F90

```diff
@@ -1,523 +1,638 @@
-         
-         currentPatch%leaf_litter(p) = currentPatch%leaf_litter(p) + leaf_litter / litter_area
-         currentPatch%root_litter(p) = currentPatch%root_litter(p) + root_litter / litter_area
-         
-         ! track as diagnostic fluxes
-         currentSite%leaf_litter_diagnostic_input_carbonflux(p) =       &
-               currentSite%leaf_litter_diagnostic_input_carbonflux(p) + &
-               leaf_litter * hlm_days_per_year / AREA
-         
-         currentSite%root_litter_diagnostic_input_carbonflux(p) =       &
-               currentSite%root_litter_diagnostic_input_carbonflux(p) + &
-               root_litter * hlm_days_per_year / AREA
-
-
-         ! Logging specific diagnostics
-         ! ----------------------------------------------------------------------------------------
-
-         ! Note that litter stock also has terms above in the CWD loop
-         delta_litter_stock  = delta_litter_stock  + &
-                               leaf_litter         + &
-                               root_litter
-
-         delta_biomass_stock = delta_biomass_stock + &
-                               leaf_litter         + &
-                               root_litter         + &
-                               (direct_dead+indirect_dead) * (struct_c + sapw_c)
-
-         delta_individual    = delta_individual    + &
-                               direct_dead         + &
-                               indirect_dead
-
-         currentCohort => currentCohort%taller
-      end do
-
-      ! Update the amount of carbon exported from the site through logging
-      ! operations.  Currently we assume only above-ground portion
-      ! of the tree bole that experienced "direct" logging is exported
-      ! This portion is known as "trunk_product_site
-
-
-      currentSite%flux_out = currentSite%flux_out + trunk_product_site
-
-      currentSite%resources_management%trunk_product_site  = &
-           currentSite%resources_management%trunk_product_site + &
-           trunk_product_site
-
-      currentSite%resources_management%delta_litter_stock  = &
-           currentSite%resources_management%delta_litter_stock + &
-           delta_litter_stock
-
-      currentSite%resources_management%delta_biomass_stock = &
-           currentSite%resources_management%delta_biomass_stock + &
-           delta_biomass_stock
-
-      currentSite%resources_management%delta_individual    = &
-           currentSite%resources_management%delta_individual + &
-           delta_individual
-
-      currentCohort => newPatch%shortest
-      do while(associated(currentCohort))
-         call carea_allom(currentCohort%dbh,currentCohort%n,currentSite%spread,currentCohort%pft,currentCohort%c_area)
-         currentCohort => currentCohort%taller
-      enddo
-
-   end subroutine logging_litter_fluxes
-
-end module EDLoggingMortalityMod
+module EDLoggingMortalityMod
```

Comment:
This seems a copy as paste of the codes. It is ok with our practice? Would you please check with Ryan on this?

Reply:
This is weird; The diff utility I'm using locally between these two branches shows discrete changes.  There are a lot of lines changed, but not the whole file.  @rgknox have you ever seen anything like this before in a PR?

Reply:
Ryan and I chatted in the hall and he suggested looking at the end of line characters.  Sure enough the file was previously using CRLF whereas this merge is using LF.  When I use the `git diff` (not the vscode app I had been using to diff), the whole file shows as different as a result.  Using `git diff --ignore-space-at-eol` it matches what I'm seeing.  I'm going to try and track down where this changed.

Reply:
Manually checking the commits, it looks like it happened within PR 524 at this commit here: https://github.com/NGEET/fates/commit/5f24b0417edb5d94e4f0468c8a59144366ffbad4#diff-135c876b8ed5286434c050b3b223605d

That said, I think it changed because I didn't have `git config --global core.autocrlf input` set in my local repo.  I'm not sure if I can fix this to show the diffs in github...

Reply:
@xuchongang I just got a github tip from Ryan that helped out immensely.  If you go to the top of this merge request and select the "ignore whitespace" and then click the apply & reload button, github will ignore the eol file changes:

![image](https://user-images.githubusercontent.com/7565064/72182086-991f3f80-339f-11ea-8a4e-f97c59104f51.png)